index.html

```diff
@@ -1608,7 +1608,31 @@ <h3>Defining 'string'</h3>
 	</div>
 </section>
 
+<section id="char_whitespace" class="subtopic">
+	<h3>Whitespace characters</h3>
+	
+	<div class="xref"><span class="seealso">See also</span>
+	<p>[[[#markup_identifiers]]].</p>
+	</div>
 
+	<p>
+		Whitespace characters are characters that represent horizontal or vertical space in typography. Different whitespace characters may have different lengths.
+	</p>
+	
+		<div class="req" id="char_define_whitespace">
+			<p class="advisement">Specifications that mention whitespace characters SHOULD explicitly state which characters are whitespace characters, by referring to the Unicode <a href="https://www.unicode.org/reports/tr44/#White_Space">White_Space</a> property or <code>General_Category=Z</code>, or specifying the specific code points.</p>
```

Comment:
Include the `[[UTR44]]` reference as a reference.

This guidance isn't quite complete. When should spec writers specify the specific code points? Which code points should be specified?

Before the mustard I would probably add a paragraph saying something like:

> Some specifications that define formal languages (examples include HTML or CSS) will choose to specify ASCII whitespace as part of their grammar. Specifications that deal more generally with text will choose to follow Unicode's definition instead.

Reply:
Fixed. Thanks.